decred_wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -37,6 +37,7 @@ class TransactionHistoryViewController: UIViewController {
         initFilterBtn()
         self.tableView.addSubview(self.refreshControl)
         self.tableView.register(UINib(nibName: TransactionTableViewCell.identifier, bundle: nil), forCellReuseIdentifier: TransactionTableViewCell.identifier)
+        self.tableView.tableFooterView = UIView.init()
```

Comment:
> its not a hack i added a footer view to it which is normal for a table view, which prevent the tableview from showing empty rows

Table view footers are not meant to "prevent the tableview from showing empty rows". Can you do this instead? It's a more straightforward solution.
> You should check if there's no data to display and remove the table view separator.

Reply:
i will take it off, seems there is a different issue created for this #317